### ISSUE TITLE
Pre-work Checklist: Developer: Richard Kwang

### ISSUE THREAD

**Issue by kwangric** [hackforla/website#4169]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas))
- [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by kwangric** [comment]
Progress: 90% done. Just need to read up on the Figma documentation.
Blockers: None
Availability: Weekdays after 1pm PDT.
ETA: Early next week
Pictures (optional): N/A

**Comment by kwangric** [comment]
Progress: Finished the checklist, currently working on my medium issue.
Blockers: Waiting on PM approval on current issue #4271 
Availability: Weekdays after 1pm PDT
ETA: 2-3 weeks, depending on the large issue
Pictures (optional): N/A

**Comment by kwangric** [comment]
Progress: Finished medium issue. Currently waiting on a large issue to start.
Blockers: Waiting for an available large issue.
Availability: Weekdays after 1pm PDT
ETA: 2-3 weeks, depending on the large issue
Pictures (optional): N/A

**Comment by kwangric** [comment]
Progress: Currently working on my large issue. Haven't had as much time to work on it this week but I'm slowly working through it.
Blockers: No blockers at the moment.
Availability: It varies since I recently started a new part time job, but I'm available most evenings through the week after 5pm PDT
ETA: Hoping to have the large issue finished by the end of next week, which should finish up the pre-work checklist.
Pictures (optional): N/A

**Comment by kwangric** [comment]
Progress: Currently working on my large issue. Making good progress and have the main functionality done, just focusing in on the details.
Blockers: No blockers at the moment.
Availability: It varies since I recently started a new part time job, but I'm available most evenings through the week after 5pm PDT
ETA: Hoping to have the large issue finished by the end of next week, which should finish up the pre-work checklist.
Pictures (optional): N/A

**Comment by kwangric** [comment]
Progress: Opened a pull request for my large issue, waiting on one more review.
Blockers: No blockers at the moment.
Availability: Most evenings through the week after 5pm PDT
ETA: Within a week
Pictures (optional): N/A

**Comment by HackforLABot** [comment]
## Activity Log: kwangric
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- kwangric opened: https://github.com/hackforla/website/issues/4169 at 03/13/2023, 08:09 PM PDT
- kwangric assigned: https://github.com/hackforla/website/issues/4169 at 03/13/2023, 08:32 PM PDT
- kwangric assigned: https://github.com/hackforla/website/issues/3953 at 03/15/2023, 01:27 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/3953#issuecomment-1470803547 at 03/15/2023, 01:35 PM PDT
- kwangric opened: https://github.com/hackforla/website/pull/4195 at 03/16/2023, 10:59 AM PDT
- kwangric PR was merged: https://github.com/hackforla/website/pull/4195#event-8773791554 at 03/16/2023, 07:59 PM PDT
- kwangric issue closed by PR 4195: https://github.com/hackforla/website/issues/3953#event-8773791632 at 03/16/2023, 07:59 PM PDT
- kwangric assigned: https://github.com/hackforla/website/issues/4145 at 03/17/2023, 10:22 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4145#issuecomment-1474168705 at 03/17/2023, 10:24 AM PDT
- kwangric opened: https://github.com/hackforla/website/pull/4202 at 03/17/2023, 10:44 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4169#issuecomment-1474688826 at 03/17/2023, 09:22 PM PDT
- kwangric PR was merged: https://github.com/hackforla/website/pull/4202#event-8790364788 at 03/19/2023, 10:30 PM PDT
- kwangric issue closed by PR 4202: https://github.com/hackforla/website/issues/4145#event-8790364937 at 03/19/2023, 10:30 PM PDT
- kwangric assigned: https://github.com/hackforla/website/issues/4220 at 03/20/2023, 09:34 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4220#issuecomment-1477269527 at 03/20/2023, 09:36 PM PDT
- kwangric opened: https://github.com/hackforla/website/pull/4224 at 03/21/2023, 10:19 AM PDT
- kwangric PR was merged: https://github.com/hackforla/website/pull/4224#event-8832634684 at 03/23/2023, 02:26 PM PDT
- kwangric issue closed by PR 4224: https://github.com/hackforla/website/issues/4220#event-8832634991 at 03/23/2023, 02:26 PM PDT
- kwangric assigned: https://github.com/hackforla/website/issues/4271 at 03/23/2023, 09:01 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4271#issuecomment-1482216958 at 03/23/2023, 09:02 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4295#issuecomment-1485683210 at 03/27/2023, 11:40 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4295#pullrequestreview-1359814667 at 03/27/2023, 01:21 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4271#issuecomment-1486020824 at 03/27/2023, 05:04 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4169#issuecomment-1492148223 at 03/31/2023, 08:35 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4271#issuecomment-1493778096 at 04/02/2023, 11:55 PM PDT
- kwangric opened: https://github.com/hackforla/website/pull/4425 at 04/04/2023, 07:41 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4424#issuecomment-1497893507 at 04/05/2023, 10:56 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4424#pullrequestreview-1375726456 at 04/06/2023, 06:10 PM PDT
- kwangric PR was merged: https://github.com/hackforla/website/pull/4425#event-8951300591 at 04/06/2023, 09:03 PM PDT
- kwangric issue closed by PR 4425: https://github.com/hackforla/website/issues/4271#event-8951300664 at 04/06/2023, 09:03 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4516#issuecomment-1509283774 at 04/14/2023, 02:18 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4516#pullrequestreview-1386607314 at 04/15/2023, 12:28 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4524#issuecomment-1510636178 at 04/16/2023, 08:18 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4524#pullrequestreview-1388982864 at 04/17/2023, 02:38 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4549#issuecomment-1516605936 at 04/20/2023, 09:15 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4549#pullrequestreview-1394375626 at 04/20/2023, 09:30 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4169#issuecomment-1520898832 at 04/24/2023, 03:15 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4576#issuecomment-1527956228 at 04/28/2023, 11:46 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4576#pullrequestreview-1406494769 at 04/28/2023, 12:34 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4584#issuecomment-1528125819 at 04/28/2023, 02:46 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4584#pullrequestreview-1406637059 at 04/28/2023, 02:50 PM PDT
- kwangric assigned: https://github.com/hackforla/website/issues/3985#issuecomment-1507419944 at 04/30/2023, 10:07 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1529082307 at 04/30/2023, 10:09 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1537502753 at 05/07/2023, 10:46 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4169#issuecomment-1546038752 at 05/12/2023, 10:05 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1546707415 at 05/13/2023, 09:42 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1555359253 at 05/19/2023, 04:04 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4169#issuecomment-1565190328 at 05/26/2023, 08:41 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/3985#issuecomment-1566307300 at 05/28/2023, 04:42 PM PDT
- kwangric opened: https://github.com/hackforla/website/pull/4764 at 05/31/2023, 07:25 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1571311760 at 05/31/2023, 09:17 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1577700117 at 06/05/2023, 04:56 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4169#issuecomment-1581143950 at 06/07/2023, 09:15 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1583430739 at 06/08/2023, 02:45 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1585495962 at 06/09/2023, 10:36 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1592092127 at 06/14/2023, 03:49 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4764#issuecomment-1592097068 at 06/14/2023, 03:54 PM PDT
- kwangric PR was merged: https://github.com/hackforla/website/pull/4764#event-9574970542 at 06/19/2023, 10:02 PM PDT
- kwangric issue closed by PR 4764: https://github.com/hackforla/website/issues/3985#event-9574970722 at 06/19/2023, 10:02 PM PDT
- kwangric closed as completed: https://github.com/hackforla/website/issues/4169#event-9581427027 at 06/20/2023, 08:52 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4936#issuecomment-1631747229 at 07/11/2023, 07:14 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4942#pullrequestreview-1528856640 at 07/13/2023, 09:42 AM PDT
- kwangric opened: https://github.com/hackforla/website/pull/4949 at 07/14/2023, 10:23 AM PDT
- kwangric closed: https://github.com/hackforla/website/pull/4949#event-9823994832 at 07/14/2023, 10:24 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1644557748 at 07/20/2023, 01:28 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5052#issuecomment-1648721320 at 07/24/2023, 03:41 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5052#pullrequestreview-1544394273 at 07/24/2023, 03:53 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5052#issuecomment-1648815868 at 07/24/2023, 06:01 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/4769#issuecomment-1650649617 at 07/25/2023, 03:29 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5061#issuecomment-1650772540 at 07/25/2023, 05:33 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5063#issuecomment-1650841439 at 07/25/2023, 06:42 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5055#issuecomment-1650874483 at 07/25/2023, 07:23 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5052#event-9929784683 at 07/26/2023, 04:45 PM PDT
- kwangric closed by PR 5052: https://github.com/hackforla/website/issues/4919#event-9929784733 at 07/26/2023, 04:45 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5063#pullrequestreview-1548782131 at 07/26/2023, 05:02 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5061#pullrequestreview-1548795534 at 07/26/2023, 05:12 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5093#issuecomment-1654559531 at 07/27/2023, 01:53 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5093#pullrequestreview-1551120738 at 07/27/2023, 06:31 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5055#pullrequestreview-1551283151 at 07/27/2023, 08:27 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/5036#issuecomment-1657247017 at 07/30/2023, 12:24 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5112#issuecomment-1658343968 at 07/31/2023, 06:09 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5117#issuecomment-1658373137 at 07/31/2023, 06:27 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5120#issuecomment-1659057527 at 07/31/2023, 01:06 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/5036#issuecomment-1659134864 at 07/31/2023, 01:43 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5117#pullrequestreview-1555884293 at 07/31/2023, 03:20 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5120#pullrequestreview-1555899966 at 07/31/2023, 03:34 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5117#event-9969813548 at 07/31/2023, 03:39 PM PDT
- kwangric closed by PR 5117: https://github.com/hackforla/website/issues/5029#event-9969813673 at 07/31/2023, 03:39 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5104#issuecomment-1659344997 at 07/31/2023, 04:29 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4942#issuecomment-1659569449 at 07/31/2023, 10:00 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5119#issuecomment-1664098728 at 08/03/2023, 07:32 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5122#issuecomment-1664303473 at 08/03/2023, 09:40 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5122#pullrequestreview-1562082694 at 08/03/2023, 07:17 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5119#pullrequestreview-1562086311 at 08/03/2023, 07:24 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5119#event-10004678373 at 08/03/2023, 07:26 PM PDT
- kwangric closed by PR 5119: https://github.com/hackforla/website/issues/4812#event-10004678467 at 08/03/2023, 07:26 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5120#issuecomment-1665905814 at 08/04/2023, 09:48 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5141#issuecomment-1666211361 at 08/04/2023, 02:47 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5145#issuecomment-1666521523 at 08/05/2023, 07:29 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5120#pullrequestreview-1564034802 at 08/05/2023, 05:05 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5145#pullrequestreview-1564043577 at 08/05/2023, 06:32 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5141#pullrequestreview-1564175826 at 08/06/2023, 09:30 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5141#event-10017594719 at 08/06/2023, 09:30 AM PDT
- kwangric closed by PR 5141: https://github.com/hackforla/website/issues/5136#event-10017594756 at 08/06/2023, 09:30 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4942#pullrequestreview-1566263155 at 08/07/2023, 04:21 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4942#pullrequestreview-1572876960 at 08/10/2023, 05:13 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5170#event-10089161917 at 08/14/2023, 09:58 AM PDT
- kwangric closed by PR 5170: https://github.com/hackforla/website/issues/5041#event-10089162113 at 08/14/2023, 09:58 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5183#event-10090297167 at 08/14/2023, 12:02 PM PDT
- kwangric closed by PR 5183: https://github.com/hackforla/website/issues/4810#event-10090297330 at 08/14/2023, 12:02 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4942#pullrequestreview-1577836963 at 08/14/2023, 05:46 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5124#issuecomment-1678286939 at 08/14/2023, 06:07 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5239#event-10121766272 at 08/17/2023, 09:57 AM PDT
- kwangric closed by PR 5239: https://github.com/hackforla/website/issues/5200#event-10121766439 at 08/17/2023, 09:57 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5203#event-10123356903 at 08/17/2023, 12:50 PM PDT
- kwangric closed by PR 5203: https://github.com/hackforla/website/issues/4248#event-10123357028 at 08/17/2023, 12:50 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/4942#pullrequestreview-1585964802 at 08/20/2023, 02:42 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/4942#issuecomment-1685567493 at 08/20/2023, 08:21 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/4942#event-10140193918 at 08/20/2023, 08:22 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5299#event-10212828508 at 08/28/2023, 12:22 PM PDT
- kwangric closed by PR 5299: https://github.com/hackforla/website/issues/5176#event-10212828679 at 08/28/2023, 12:22 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5296#issuecomment-1696249697 at 08/28/2023, 12:24 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5298#issuecomment-1701068235 at 08/31/2023, 06:44 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5298#pullrequestreview-1616366071 at 09/07/2023, 04:28 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5522#event-10386087695 at 09/15/2023, 10:54 AM PDT
- kwangric closed by PR 5522: https://github.com/hackforla/website/issues/5409#event-10386087885 at 09/15/2023, 10:54 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5538#issuecomment-1723534568 at 09/18/2023, 07:25 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5537#issuecomment-1723535949 at 09/18/2023, 07:25 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5550#issuecomment-1723536894 at 09/18/2023, 07:26 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5551#issuecomment-1723548640 at 09/18/2023, 07:31 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5551#issuecomment-1723603259 at 09/18/2023, 07:49 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5537#pullrequestreview-1631451736 at 09/18/2023, 09:09 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5538#pullrequestreview-1631455358 at 09/18/2023, 09:11 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5550#pullrequestreview-1631478015 at 09/18/2023, 09:22 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5537#event-10402269232 at 09/18/2023, 12:47 PM PDT
- kwangric closed by PR 5537: https://github.com/hackforla/website/issues/5429#event-10402269416 at 09/18/2023, 12:47 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5543#event-10405162334 at 09/18/2023, 07:59 PM PDT
- kwangric closed by PR 5543: https://github.com/hackforla/website/issues/5426#event-10405162418 at 09/18/2023, 07:59 PM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5556#pullrequestreview-1633461002 at 09/19/2023, 07:32 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5538#pullrequestreview-1633642901 at 09/19/2023, 08:51 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5538#event-10423741019 at 09/20/2023, 07:03 AM PDT
- kwangric closed by PR 5538: https://github.com/hackforla/website/issues/5341#event-10423741389 at 09/20/2023, 07:03 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5413#event-10423758426 at 09/20/2023, 07:05 AM PDT
- kwangric closed by PR 5413: https://github.com/hackforla/website/issues/4456#event-10423758775 at 09/20/2023, 07:05 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5298#pullrequestreview-1638888704 at 09/21/2023, 05:02 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5585#event-10491352930 at 09/27/2023, 12:21 PM PDT
- kwangric closed by PR 5585: https://github.com/hackforla/website/issues/5388#event-10491353101 at 09/27/2023, 12:21 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5576#event-10512430444 at 09/29/2023, 07:57 AM PDT
- kwangric closed by PR 5576: https://github.com/hackforla/website/issues/5508#event-10512430685 at 09/29/2023, 07:57 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5565#event-10512470451 at 09/29/2023, 08:01 AM PDT
- kwangric closed by PR 5565: https://github.com/hackforla/website/issues/5505#event-10512470695 at 09/29/2023, 08:01 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5551#event-10513946605 at 09/29/2023, 10:42 AM PDT
- kwangric closed by PR 5551: https://github.com/hackforla/website/issues/5342#event-10513946797 at 09/29/2023, 10:42 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5551#issuecomment-1741271482 at 09/29/2023, 10:42 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/5391#issuecomment-1743447532 at 10/02/2023, 10:26 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/5371#issuecomment-1746026181 at 10/03/2023, 07:18 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5648#issuecomment-1746945446 at 10/04/2023, 07:03 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5646#event-10551610815 at 10/04/2023, 07:04 AM PDT
- kwangric closed by PR 5646: https://github.com/hackforla/website/issues/5366#event-10551611808 at 10/04/2023, 07:04 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5645#issuecomment-1746979024 at 10/04/2023, 07:20 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5648#pullrequestreview-1657708072 at 10/04/2023, 07:39 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5645#event-10556854808 at 10/04/2023, 05:15 PM PDT
- kwangric closed by PR 5645: https://github.com/hackforla/website/issues/5447#event-10556854938 at 10/04/2023, 05:15 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5671#issuecomment-1751777096 at 10/07/2023, 10:30 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5672#event-10593815488 at 10/09/2023, 10:21 AM PDT
- kwangric closed by PR 5672: https://github.com/hackforla/website/issues/5624#event-10593815653 at 10/09/2023, 10:21 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5668#issuecomment-1753618198 at 10/09/2023, 12:43 PM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/5504#issuecomment-1764918439 at 10/16/2023, 10:11 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5718#issuecomment-1765102874 at 10/16/2023, 11:59 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5718#pullrequestreview-1680793130 at 10/16/2023, 12:13 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5719#event-10693815348 at 10/18/2023, 07:19 AM PDT
- kwangric closed by PR 5719: https://github.com/hackforla/website/issues/5675#event-10693815655 at 10/18/2023, 07:19 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5732#issuecomment-1772759144 at 10/20/2023, 06:39 AM PDT
- kwangric submitted review: https://github.com/hackforla/website/pull/5732#pullrequestreview-1690090587 at 10/20/2023, 06:50 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5732#event-10724811238 at 10/20/2023, 06:51 AM PDT
- kwangric closed by PR 5732: https://github.com/hackforla/website/issues/5370#event-10724811503 at 10/20/2023, 06:51 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5731#issuecomment-1775962166 at 10/23/2023, 01:23 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5731#event-10746017651 at 10/23/2023, 01:38 PM PDT
- kwangric closed by PR 5731: https://github.com/hackforla/website/issues/5371#event-10746017830 at 10/23/2023, 01:38 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5768#event-10761421339 at 10/24/2023, 07:33 PM PDT
- kwangric closed by PR 5768: https://github.com/hackforla/website/issues/5713#event-10761421514 at 10/24/2023, 07:33 PM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5771#event-10782519415 at 10/26/2023, 09:55 AM PDT
- kwangric closed by PR 5771: https://github.com/hackforla/website/issues/5763#event-10782519689 at 10/26/2023, 09:55 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/5595#issuecomment-1782727382 at 10/27/2023, 04:08 AM PDT
- kwangric commented on: https://github.com/hackforla/website/issues/5528#issuecomment-1783270420 at 10/27/2023, 10:30 AM PDT
- kwangric closed as completed: https://github.com/hackforla/website/issues/5528#event-10794196162 at 10/27/2023, 10:30 AM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5795#issuecomment-1786004274 at 10/30/2023, 01:41 PM PDT
- kwangric closed: https://github.com/hackforla/website/pull/5795#event-10812851649 at 10/30/2023, 01:42 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5820#issuecomment-1790750304 at 11/02/2023, 06:38 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5794#event-10850945368 at 11/02/2023, 07:36 PM PDT
- kwangric closed by PR 5794: https://github.com/hackforla/website/issues/5753#event-10850945430 at 11/02/2023, 07:36 PM PDT
- kwangric commented on: https://github.com/hackforla/website/pull/5828#issuecomment-1792921143 at 11/03/2023, 11:17 AM PDT
- kwangric merged: https://github.com/hackforla/website/pull/5853#event-10876695382 at 11/06/2023, 10:05 AM PST
- kwangric closed by PR 5853: https://github.com/hackforla/website/issues/5852#event-10876695636 at 11/06/2023, 10:05 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5859#issuecomment-1796226395 at 11/06/2023, 11:49 AM PST
- kwangric closed: https://github.com/hackforla/website/pull/5859#event-10877836264 at 11/06/2023, 11:49 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5835#issuecomment-1802055302 at 11/08/2023, 06:56 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/5877#event-10907349916 at 11/08/2023, 05:09 PM PST
- kwangric closed by PR 5877: https://github.com/hackforla/website/issues/5856#event-10907350038 at 11/08/2023, 05:09 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5884#pullrequestreview-1724930913 at 11/10/2023, 06:39 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5886#pullrequestreview-1724966251 at 11/10/2023, 06:57 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/5886#event-10926154946 at 11/10/2023, 06:57 AM PST
- kwangric closed by PR 5886: https://github.com/hackforla/website/issues/5755#event-10926155191 at 11/10/2023, 06:58 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5884#issuecomment-1806216857 at 11/10/2023, 10:28 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5894#issuecomment-1809122127 at 11/13/2023, 01:06 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5835#event-10947109613 at 11/13/2023, 01:08 PM PST
- kwangric closed by PR 5835: https://github.com/hackforla/website/issues/5760#event-10947109839 at 11/13/2023, 01:08 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5894#event-10961867447 at 11/14/2023, 05:36 PM PST
- kwangric closed by PR 5894: https://github.com/hackforla/website/issues/5752#event-10961867551 at 11/14/2023, 05:36 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5905#event-10969733425 at 11/15/2023, 08:39 AM PST
- kwangric closed by PR 5905: https://github.com/hackforla/website/issues/5652#event-10969733774 at 11/15/2023, 08:39 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5911#pullrequestreview-1738001563 at 11/17/2023, 01:56 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5911#event-10999505098 at 11/17/2023, 01:56 PM PST
- kwangric closed by PR 5911: https://github.com/hackforla/website/issues/5855#event-10999505285 at 11/17/2023, 01:56 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5907#event-10999511384 at 11/17/2023, 01:57 PM PST
- kwangric closed by PR 5907: https://github.com/hackforla/website/issues/5712#event-10999511555 at 11/17/2023, 01:57 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5908#event-11015418984 at 11/20/2023, 10:40 AM PST
- kwangric closed by PR 5908: https://github.com/hackforla/website/issues/5770#event-11015419180 at 11/20/2023, 10:40 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/5922#event-11015548185 at 11/20/2023, 10:57 AM PST
- kwangric closed by PR 5922: https://github.com/hackforla/website/issues/5921#event-11015548399 at 11/20/2023, 10:57 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5914#issuecomment-1819823289 at 11/20/2023, 01:22 PM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5932#issuecomment-1823533431 at 11/22/2023, 01:28 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5927#event-11041390588 at 11/22/2023, 01:29 PM PST
- kwangric closed by PR 5927: https://github.com/hackforla/website/issues/5919#event-11041390735 at 11/22/2023, 01:29 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5925#event-11041395757 at 11/22/2023, 01:30 PM PST
- kwangric closed by PR 5925: https://github.com/hackforla/website/issues/5804#event-11041395869 at 11/22/2023, 01:30 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5918#event-11041437759 at 11/22/2023, 01:36 PM PST
- kwangric closed by PR 5918: https://github.com/hackforla/website/issues/5857#event-11041437878 at 11/22/2023, 01:36 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5915#event-11041525053 at 11/22/2023, 01:51 PM PST
- kwangric closed by PR 5915: https://github.com/hackforla/website/issues/5589#event-11041525158 at 11/22/2023, 01:52 PM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5914#issuecomment-1823558398 at 11/22/2023, 01:55 PM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5914#issuecomment-1823626605 at 11/22/2023, 03:32 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5914#event-11065015373 at 11/26/2023, 05:55 PM PST
- kwangric closed by PR 5914: https://github.com/hackforla/website/issues/5807#event-11065015482 at 11/26/2023, 05:55 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5932#pullrequestreview-1756456870 at 11/29/2023, 04:52 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5945#event-11110123884 at 11/30/2023, 06:51 AM PST
- kwangric closed by PR 5945: https://github.com/hackforla/website/issues/5511#event-11110124349 at 11/30/2023, 06:51 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/5940#event-11110315569 at 11/30/2023, 07:05 AM PST
- kwangric closed by PR 5940: https://github.com/hackforla/website/issues/5706#event-11110315890 at 11/30/2023, 07:06 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5932#pullrequestreview-1760780333 at 12/01/2023, 09:03 PM PST
- kwangric commented on: https://github.com/hackforla/website/issues/5676#issuecomment-1837177555 at 12/02/2023, 07:09 AM PST
- kwangric closed: https://github.com/hackforla/website/pull/5954#event-11129178717 at 12/02/2023, 07:10 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5952#issuecomment-1837178326 at 12/02/2023, 07:11 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/5951#issuecomment-1837178576 at 12/02/2023, 07:12 AM PST
- kwangric commented on: https://github.com/hackforla/website/issues/5676#issuecomment-1837204088 at 12/02/2023, 09:05 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5952#pullrequestreview-1760937319 at 12/02/2023, 10:38 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/5952#event-11129555330 at 12/02/2023, 10:38 AM PST
- kwangric closed by PR 5952: https://github.com/hackforla/website/issues/5716#event-11129555357 at 12/02/2023, 10:38 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5951#pullrequestreview-1760937640 at 12/02/2023, 10:41 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/5951#event-11129560741 at 12/02/2023, 10:42 AM PST
- kwangric closed by PR 5951: https://github.com/hackforla/website/issues/5798#event-11129560787 at 12/02/2023, 10:42 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/5932#pullrequestreview-1766295798 at 12/05/2023, 03:31 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/5950#event-11201318335 at 12/09/2023, 11:38 AM PST
- kwangric closed by PR 5950: https://github.com/hackforla/website/issues/5674#event-11201318363 at 12/09/2023, 11:38 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/6004#issuecomment-1854396749 at 12/13/2023, 09:16 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/6003#event-11260386234 at 12/15/2023, 06:42 AM PST
- kwangric closed by PR 6003: https://github.com/hackforla/website/issues/5819#event-11260386456 at 12/15/2023, 06:42 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/6010#event-11282377258 at 12/18/2023, 02:36 PM PST
- kwangric closed by PR 6010: https://github.com/hackforla/website/issues/5684#event-11282377437 at 12/18/2023, 02:36 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/6004#event-11303743267 at 12/20/2023, 06:59 AM PST
- kwangric closed by PR 6004: https://github.com/hackforla/website/issues/5848#event-11303743606 at 12/20/2023, 06:59 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/5942#event-11380099088 at 01/03/2024, 06:58 AM PST
- kwangric closed by PR 5942: https://github.com/hackforla/website/issues/5333#event-11380099279 at 01/03/2024, 06:58 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6056#pullrequestreview-1804410886 at 01/04/2024, 07:30 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/6056#event-11391428150 at 01/04/2024, 07:31 AM PST
- kwangric closed by PR 6056: https://github.com/hackforla/website/issues/5934#event-11391428410 at 01/04/2024, 07:31 AM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6076#pullrequestreview-1810061065 at 01/08/2024, 01:33 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6077#pullrequestreview-1810064070 at 01/08/2024, 01:35 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6075#pullrequestreview-1810071916 at 01/08/2024, 01:40 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6073#pullrequestreview-1810076193 at 01/08/2024, 01:44 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/6073#event-11422864810 at 01/08/2024, 01:45 PM PST
- kwangric closed by PR 6073: https://github.com/hackforla/website/issues/5958#event-11422864949 at 01/08/2024, 01:45 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6068#pullrequestreview-1810083634 at 01/08/2024, 01:50 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/6068#event-11422909322 at 01/08/2024, 01:50 PM PST
- kwangric closed by PR 6068: https://github.com/hackforla/website/issues/6019#event-11422909463 at 01/08/2024, 01:50 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/6098#event-11491882225 at 01/15/2024, 12:23 PM PST
- kwangric closed by PR 6098: https://github.com/hackforla/website/issues/5830#event-11491882342 at 01/15/2024, 12:23 PM PST
- kwangric commented on: https://github.com/hackforla/website/pull/6099#issuecomment-1892743999 at 01/15/2024, 12:42 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6099#pullrequestreview-1822245810 at 01/15/2024, 12:54 PM PST
- kwangric submitted review: https://github.com/hackforla/website/pull/6132#pullrequestreview-1832915046 at 01/19/2024, 06:51 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/6132#event-11538531576 at 01/19/2024, 06:51 AM PST
- kwangric closed by PR 6132: https://github.com/hackforla/website/issues/6106#event-11538531999 at 01/19/2024, 06:51 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/6135#issuecomment-1904867655 at 01/22/2024, 01:40 PM PST
- kwangric commented on: https://github.com/hackforla/website/pull/6133#issuecomment-1904871279 at 01/22/2024, 01:42 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/6133#event-11591486840 at 01/24/2024, 05:39 PM PST
- kwangric closed by PR 6133: https://github.com/hackforla/website/issues/6064#event-11591486939 at 01/24/2024, 05:39 PM PST
- kwangric merged: https://github.com/hackforla/website/pull/6207#event-11693315163 at 02/03/2024, 11:10 AM PST
- kwangric closed by PR 6207: https://github.com/hackforla/website/issues/6046#event-11693315188 at 02/03/2024, 11:10 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/6210#event-11693334662 at 02/03/2024, 11:23 AM PST
- kwangric closed by PR 6210: https://github.com/hackforla/website/issues/6047#event-11693334697 at 02/03/2024, 11:23 AM PST
- kwangric commented on: https://github.com/hackforla/website/pull/6210#issuecomment-1925435456 at 02/03/2024, 11:25 AM PST
- kwangric merged: https://github.com/hackforla/website/pull/6214#event-11709057210 at 02/05/2024, 11:39 AM PST
- kwangric closed by PR 6214: https://github.com/hackforla/website/issues/6048#event-11709057474 at 02/05/2024, 11:39 AM PST
